packages/kbn-management/settings/application/services.tsx

```diff
@@ -6,17 +6,37 @@
  * Side Public License, v 1.
  */
 
-import React, { FC } from 'react';
+import React, { FC, useContext } from 'react';
 
 import {
   FormProvider,
   FormKibanaProvider,
   type FormKibanaDependencies,
   type FormServices,
 } from '@kbn/management-settings-components-form';
+import { UiSettingMetadata } from '@kbn/management-settings-types';
+import { IUiSettingsClient } from '@kbn/core-ui-settings-browser';
+import { normalizeSettings } from '@kbn/management-settings-utilities';
+import { Subscription } from 'rxjs';
 
-export type SettingsApplicationServices = FormServices;
-export type SettingsApplicationKibanaDependencies = FormKibanaDependencies;
+export interface Services {
```

Comment:
These are the actual methods we're using in the application.

By abstracting away the Kibana dependencies in favor of distinct methods, we can actually mock them in Jest or Storybook without having to implement the entire `SettingsStart` contract.  It also makes it easier to implement them directly in code, (for cases like caching or state, where we don't want to run the method directly against the dependent contract every time).